lib/ykpiv.c

```diff
@@ -202,6 +203,53 @@ ykpiv_rc ykpiv_connect(ykpiv_state *state, const char *wanted) {
   return YKPIV_GENERIC_ERROR;
 }
 
+ykpiv_rc ykpiv_reconnect(ykpiv_state *state) {
```

Comment:
Please make this function static and not part of the ykpiv_ namespace (or if we really want it to be part of the API add it to the map file for exporting as well).